src/plots/cartesian/set_convert.js

```diff
@@ -381,9 +381,6 @@ module.exports = function setConvert(ax, fullLayout) {
         }
 
         if(!isFinite(ax._m) || !isFinite(ax._b)) {
-            Lib.notifier(
```

Comment:
Let me guess: you removed this so that we don't have to pass `gd` to `setConvert` ?

Regardless, I'm ok with making this just an (English-only) error :wink: